src/DraftJSEditor.jsx

```diff
@@ -349,13 +351,33 @@ class DraftJSEditor extends Component {
       ]);
     }
 
+    const passedProps = this.props;
+
+    const cleanObject = (object, keyArray) => {
```

Comment:
This feels like something that would be a good idea to break out into its own file, something like `src/utils/cleanObject.js`